apr-util.yaml

```diff
@@ -0,0 +1,77 @@
+package:
+  name: apr-util
+  version: 1.6.3
+  epoch: 0
+  description: The Apache Portable Runtime Utility Library
+  copyright:
+    # Some files also contain RSA-MD.
+    # See https://www.ietf.org/ietf-ftp/ietf/IPR/RSA-MD-all
+    # for details on using it without license, i.e. under Apache-2.0 in this case.
+    - license: Apache-2.0 AND Public-Domain AND CC0-1.0 AND OLDAP-2.7 AND Beerware
```

Comment:
```suggestion
    - license: Apache-2.0
```

Reply:
Shouldn't the license list be complete?

Reply:
we use the license of the source package similar to arch linux and alpine , the LICENSE file in the apr-utils refers as Apache-2.0 

 @imjasonh , @kaniini  : for further clarification

Reply:
I had a chat with @imjasonh , list of license you have seems to be good